include/SSVOpenHexagon/Components/CPlayer.hpp

```diff
@@ -31,6 +31,7 @@ class CPlayer
     float size;
     float speed;
     float focusSpeed;
+    float* playerSpeedMult;
```

Comment:
Any reason why you designed this PR the way it is? It seems unusual that the player object itself holds a pointer to its speed multiplayer (see below...)